projects/Mallard/src/services/logging.ts

```diff
@@ -36,6 +36,14 @@ interface LogParams {
 type QueryData = { gdprAllowPerformance: GdprSwitchSetting }
 const QUERY = gql('{ gdprAllowPerformance @client }')
 
+const cropMessage = (
+    message: string,
+    maxLength: number,
+    overflowMessage = '... (message cropped)',
```

Comment:
This seems unnecessary as a parameter?

Reply:
it made it easier to test, but I'm happy to remove